powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationClaimRequest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2024 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.security.powerauth.client.model.request;
+
+import lombok.Data;
+
+/**
+ * Request for operation claim.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Data
+public class OperationClaimRequest  {
+
+    /**
+     * Operation identifier.
+     */
+    private String operationId;
+
+    /**
+     * Optional user identifier of the user who is claiming the operation.
```

Comment:
🤔 Here it is written _optional_, but required in `OperationClaimRequestValidator`. Could we use the bean validator or do you prefer to be consistent with other use cases?

Reply:
I will fix the wording. We can move to the bean validator, but lets do it as another issue for all validations.